### DISCUSSION TITLE
🔗 🚀 Feature Request: Hide Empty Taxonomy Based on Current Query (Duplicate of #23418)

### DISCUSSION THREAD

**Discussion by yyc-food-blog** [elementor/elementor#23392] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I like the taxonomy filter for loop grids feature; however, it would be great if you chose to hide empty taxonomies that it would hide the empty ones for the query, not just ones that are empty overall. I'm getting a lot of filters showing up that are empty for the specific query.   

Secondly, I'm using the scroll horizontal feature for the taxonomies for mobile phones. However, it's not obvious that users can scroll. Would be nice if arrows or some of other visual signal could be included to prompt the user to scroll for more filters. 

### Describe the solution you'd like

When you hide empty taxonomies, it hides ones that are empty for a specific query. Also, would be nice to manually be able to hide some categories. For example, if I have a query for the category "reviews", I would only like to show taxonomies that are children of that category, not every single category I have on the website.

Add arrows or some sort of visual cue that makes it easier for users to see they can horizontally scroll through taxonomies.

**Comment by nicholaszein** [comment]
Hello there, @yyc-food-blog!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #23418

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

----

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

Please follow our guidelines for submitting a new [🚀 Feature Request](https://github.com/elementor/elementor/discussions/new?category=feature-request), and submit only ☝️ **_<ins>one request per thread</ins>_** so we can track them better and allow users to vote only for the features they want to see implemented.

In this case, you should submit a separate request for the Horizontal Scroll feature of the Taxonomy Filter.

Kind regards
